pypeit/par/pypeitpar.py

```diff
@@ -836,7 +836,7 @@ def __init__(self, spec_method=None, spec_maxshift=None, spectrum=None,
                           'Options are: None, {0}'.format(', '.join(options['spec_method']))
 
         defaults['spec_maxshift'] = 20
-        dtypes['spec_maxshift'] = [int, float]
+        dtypes['spec_maxshift'] = int
```

Comment:
@kbwestfall said: Similar comment as above. Need input from someone who knows the flexure code better.

Reply:
Yes, apparently this needs to be only integer. We never caught it before. Thanks @tbowers7

Reply:
If one were to specify a `float` value of `spec_maxshift`, the code crashes with the following error:
```
  File "/Users/tbowers/d1/codes/PypeIt/pypeit/core/flexure.py", line 256, in spec_flex_shift
    max_corr = np.argmax(corr[lag0-mxshft:lag0+mxshft]) + lag0-mxshft
TypeError: slice indices must be integers or None or have an __index__ method
```